fmriprep/workflows/bold.py

```diff
@@ -683,9 +684,7 @@ def init_bold_hmc_wf(metadata, bold_file_size_gb, ignore,
     hmc = pe.Node(fsl.MCFLIRT(save_mats=True, save_plots=True),
                   name='BOLD_hmc', mem_gb=bold_file_size_gb * 3)
 
-    hcm2itk = pe.MapNode(c3.C3dAffineTool(fsl2ras=True, itk_transform=True),
-                         iterfield=['transform_file'], name='hcm2itk',
-                         mem_gb=0.05)
+    hcm2itk = pe.Node(MCFLIRT2ITK(), name='hcm2itk', mem_gb=0.05)
```

Comment:
@effigies what would you say it is best: setting `n_procs` here to `omp_nthreads` or leaving it serial? Locally I see the linear improvement with parallelism and I'd expect large improvements for very long BOLD files. WDYT?

Reply:
Yeah, parallelizing sounds good. (FYI I haven't had a chance to review this in detail, yet.)